app/db/database.py

```diff
@@ -0,0 +1,46 @@
+import os
+import sys
+
+from sqlalchemy import Column, ForeignKey, Integer, String, Text, Boolean
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship, sessionmaker
+from sqlalchemy import create_engine
+
+Base = declarative_base()
+
+
+class Employees(Base):
+    __tablename__ = "employees"
+    emp_name = Column(String(32), primary_key=True, nullable=False)
+    emp_role = Column(String(32), nullable=False)
+    # office_allocation = Column(String(10))
+    # living_allocation = Column(String(10))
+
```

Comment:
Remove commented code. Using the name as your primary key is bit shaky. Consider an autoincrementing integer id or a uuid